vk/backend/backend.go

```diff
@@ -146,6 +149,27 @@ func NewBackend(ctx context.Context, rp runtimeTypes.ContainerRuntimeProvider, p
 		return nil, errors.New("error converting network resource units")
 	}
 
+	kernelVersionRaw, err := os.ReadFile("/proc/sys/kernel/osrelease")
```

Comment:
Can you make all the logic for computing the value (CSV) for the runtime versions into a function?